packages/api/bin/cli.js

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+/* eslint-disable node/shebang */
```

Comment:
Do our lint rules need updating? I saw this fail lint locally as well but CI passes.

Reply:
Not sure - I generally rely on my IDE linter, I hadn't noticed lint was ok on CI.    Possibly.    It's also possible I'm not loading the lint rules correctly.

Reply:
Yeah, there does seem to be a mismatch. I've only seen it on this one rule in this one instance though. I'm ok disabling the rule manually here.